src/applications/check-in/day-of/pages/TravelQuestion.jsx

```diff
@@ -49,12 +49,28 @@ const TravelQuestion = props => {
         </li>
         <li>
           <Trans
-            i18nKey="youre-traveling-in-relation-to-a-compensation-and-pension"
+            i18nKey="you-cant-afford-to-pay-for-your-travel"
             components={[
               <span key="bold" className="vads-u-font-weight--bold" />,
             ]}
           />
         </li>
+        <li>
+          <Trans
+            i18nKey="youre-traveling-for-one-of-these-reasons"
+            components={[
+              <span key="bold" className="vads-u-font-weight--bold" />,
+            ]}
+          />
+          <p>
```

Comment:
You could add a margin bottom of zero here to get rid of the extra space after the paragraph `vads-u-margin-bottom--0` I think

Reply:
Sorry one last nitpick I noticed after I approved the first time